CIP-1694/README.md

```diff
@@ -0,0 +1,1161 @@
+* a 'Yes'/'No'/'Abstain' vote
+
+For SPOs and DReps, the number of votes that are cast (whether 'Yes', 'No' or 'Abstain') is proportional to the Lovelace that is delegated to them at the point the
+action is checked for ratification.  For constitututional committee members, each current committee member has one vote.
+Votes from unregistered SPOs or DReps count as having zero stake.
+
+> **Warning** 'Abstain' votes are not included in the "active voting stake".
+>
+> Note that an explicit vote to abstain differs from abstaining from voting.
+> Unregistered stake that did not vote behaves like an 'Abstain' vote,
+> while registered stake that did not vote behaves like a 'No' vote.
+> To avoid confusion, we will only use the word 'Abstain' from this point onward to mean an on-chain vote to abstain.
+
+The governance credential witness will trigger the appropriate verifications in the ledger according to the existing `UTxOW` ledger rule
+(i.e. a signature check for verification keys, and a validator execution with a specific vote redeemer and new Plutus purpose for scripts).
+
+Votes can be cast multiple times for each governance action by a single credential.
+Correctly submitted votes override any older votes for the same credential and role.
+That is, the voter may change their position on any action if they choose.
+As soon as a governance action is ratified, voting ends and transactions containing further votes are invalid.
+
+#### Governance state
+
+When a governance action is successfully submitted to the chain, its progress will be tracked by the ledger state.
+In particular, the following will be tracked:
+
+* the governance action ID
+* the epoch that the action expires
+* the deposit amount
+* the rewards address that will receive the deposit when it is returned
+* the total 'Yes'/'No'/'Abstain' votes of the constitutional committee for this action
+* the total 'Yes'/'No'/'Abstain' votes of the DReps for this action
+* the total 'Yes'/'No'/'Abstain' votes of the SPOs  for this action
+
+
+#### Changes to the stake snapshot
+
+Since the stake snapshot changes at each epoch boundary, a new tally must be calculated when each unratified governance action
+is checked for ratification.  This means that an action could be enacted even though the DRep or SPO votes have not changed
+(since the vote delegation could have changed).
+
+#### Definitions relating to voting stake
+
+We define a number of new terms related to voting stake:
+
+* Lovelace contained in a transaction output is considered **active for voting** (that is, it forms the "active voting stake"):
+  * It contains a registered stake credential.
+  * The registered stake credential has delegated its voting rights to a registered DRep.
+* Relative to some percentage `P`, a DRep (SPO) **vote threshold has been met** if the sum of the relative stake that has been delegated to the DReps (SPOs)
+  that vote 'Yes' to a governance action
+  is at least `P`.
```

Comment:
Does this line imply something other than `DRep or SPO **voted threshold has not been met**` or `DReps and SPOs that vote 'Yes'`? If not then I suggest:

```suggestion
* Relative to some percentage `P`, a DRep or SPO **vote threshold has been met** if the sum of the relative stake that has been delegated to the DReps or SPOs
  that vote 'Yes' to a governance action
  is at least `P`.
```

Reply:
The trouble is the ambiguity of using logical connectives is plain English. (though in this instance I think the intent is clear).

"A or B implies C or D" can sometimes mean "(A or B) -> (C or D)". which is clearly not what we intend. It is a common idiom to use "A (B) implies C (D)" as short hand for "(A -> C) and (B -> D)".

Reply:
@JaredCorduan Apologies on re-opening the topic. While I do agree that plain English does not help in this situation I also think this block actually demonstrates your argument on that topic and could still be improved for the reader. 

I don't suspect any of this will be unfamiliar but still think its application in MD or RST formats should be considered, especially when writing about complex topics. 

-------

**Common practices when writing arithmetic operations combined with text**

* Follow the general rules of grammar
  * Write in full sentences, even with arithmetic operations.
  * Do not use irregular abbreviations or shorthand forms which do not conform to standard writing conventions.
* Variables in equations or functions are usually italicized, to set them apart from text.

-------

Various tools which aid in producing arithmetic notation to be included in PDF's, whitepapers, etc. tend to follow these or similar rules/conventions. Many of these tools use specific fonts, pre-formatted/fixed font text or simply italicize the entire arithmetic "sentence" when not using distinct code blocks to separate the algorithm.  For example this was done with `P`, but not for `(SPO)` or `DRep (SPO)`.

I suggest at minimum the italicizing of arithmetic variables **especially** when the variable is also used as a bare word throughout the document in non arithmetic contexts. 
* DRep appears 123 times
  * DReps appears 65 times 
* SPO 37 times
* () are used over 100 times throughout the document
  * Over 90% of the usage is in bare grammar, not arithmetic context/examples.

Parenthesis are also non arithmetic symbols used in grammar. So wrapping a bare (non italicized) word in parenthesis I don't see as a indicative of arithmetic only examples, even if otherwise the example could be considered "complete mathematical sentence".

Reply:
Maybe it's my interpretation of your first response as it appeared you were using arithmetic explanations for your idiom to imply something. Also your example is included in many of the types of papers I mention, and in discussions and lectures once they are written, whether it be whiteboard or paper, these standards are generally applied by the professor, students, and/or authors as I mentioned above.

My point was simply to be explicit vs. implying anything, which is then left to the readers interpretation, my apologies for any of my own misinterpretations.